apps/aws-notebook/samples/ecs/fargate/vpn/README.md

```diff
@@ -0,0 +1,38 @@
+# Scripts to quicky deploy a aws-notebook  development environment
+This will create a AWS ECS cluster and a aws-notebook sandbox app by some commands.
```

Comment:
Insert an empty line between title and paragraph.

- `a AWS` => `an AWS`
- `a aws-notebook` => `an aws-notebook`
- `by some commands` => remove